test/unit/exception_handling/exception_info_test.rb

```diff
@@ -261,6 +279,23 @@ class ExceptionInfoTest < ActiveSupport::TestCase
 
         assert_equal_with_diff expected_data, prepare_data(exception_info.enhanced_data)
       end
+
+      should "log info if the custom data hook results in a nil message exception" do
+        ExceptionHandling.custom_data_hook = lambda do |data|
+          exception_with_nil_message = RuntimeError.new(nil)
+          stub(exception_with_nil_message).message { nil }
+          raise exception_with_nil_message
```

Comment:
These 3 lines are used in `exception_handling_test` as well for custom data. Helper method for it?